code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -172,7 +174,9 @@
 		create_shrapnel(U,1,0,0,shrapnel_type,cause_data,FALSE,100) //simulates a bullet
 		for(var/atom/movable/AM in U)
 			if(iscarbon(AM))
+				var/mob/living/M = AM
```

Comment:
Probably want to cast this to carbon rather than living. No single letter vars so rename as well.